### ISSUE TITLE
Vivint cameras won’t stream in Homekit

### ISSUE THREAD

**Issue by rob74lee** [daymondm/homebridge-vivint#5]
Good afternoon,

My Vivint cameras will no longer stream in HomeKit. After checking the logs it seems to create the following error: frmpeg exited with code 1. Any idea what this could be? It sounds like a ffmpeg issue but my nest cam works flawlessly. Any help would be greatly appreciated.

**Comment by daymondm** [comment]
I usually get that error when I try to view the camera stream from the same computer that ffmpeg is running on. You might check and see if the IP address for your Vivint panel has changed too

**Comment by daymondm** [comment]
Add “debug”: true to the videoconfig section in your config file for one of the camera streams, this should dump extra debugging information into the terminal/log file to get to the bottom of the error

**Comment by daymondm** [comment]
any luck @rob74lee ?

**Comment by rob74lee** [comment]
@daymondm  nah. I literally just gave up and dropped those cameras in rooms and turned off favorites for them.

**Comment by jeffmcnabb** [comment]
Did Camera streams end up getting removed? I have not been able to get any of mine to stream video; they only show up as PIV Motion. Is there some type of extra config needed?

**Comment by balansse** [comment]
@rob74lee @jeffmcnabb check out my [fork](https://github.com/balansse/homebridge-vivint), in version 0.0.10 I added a config parameter to simplify adding cameras to the ffmpeg plugin. All you'll need to do is run the plugin with a showCameraConfig flag enabled in config - valid config file for each camera will be displayed in the log.
